### PR TITLE
Administer ekg

### PR DESCRIPTION
**Pull request by roched1atwit**
Finished the page for how to administer an EKG

### CODE REVIEW

**Comment by roched1atwit** [comment]
Also in looking at the other merge requests Im seeing that we are  adding a citation at the bottom of the page from where the info came from, ill add that

File: src/css/how-to-place.css [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+    border-color: #3f3f3f;
+    border-radius: 0.5rem;
+    background-color: #dddddd;
+    text-align:center;
+}
+
+h1 {
+    font-size: 2rem;
+}
+
```
Comment (wintleh) [inline src/css/how-to-place.css]: This should help set the header sizes.

```suggestion

#content-wrapper h1 {
    font-size: 2rem;
}

#content-wrapper h2 {
    font-size: 1.6rem;
}

```

File: src/how-to-place.php [hunk 2]
```diff
@@ -1,17 +1,189 @@
...
+                <p>
+                    <ul>
+                        <li>
+                            <a href="https://www.ausmed.com/cpd/articles/ecg-lead-placement">https://www.ausmed.com/cpd/articles/ecg-lead-placement</a>
+                        </li>
+                        <li>
+                            <a href="https://www.primemedicaltraining.com/12-lead-ecg-placement/">https://www.primemedicaltraining.com/12-lead-ecg-placement/<a>
+                        </li>
+                    </ul>
+                </p>
```
Comment (wintleh) [inline src/how-to-place.php]: This should help with the spacing. The paragraph tag is being styled with a padding, and its not really necessary here.

```suggestion
                    <ul>
                        <li>
                            <a href="https://www.ausmed.com/cpd/articles/ecg-lead-placement">https://www.ausmed.com/cpd/articles/ecg-lead-placement</a>
                        </li>
                        <li>
                            <a href="https://www.primemedicaltraining.com/12-lead-ecg-placement/">https://www.primemedicaltraining.com/12-lead-ecg-placement/<a>
                        </li>
                    </ul>

```

File: src/css/how-to-place.css [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+    text-align: center;
+    font-size: 140%;
+}
+
+#page-references header {
+    padding: 1vh 2vw 1vh 2vw;
+    text-align: center;
+    font-size: 140%;
+}
+
```
Comment (wintleh) [inline src/css/how-to-place.css:75]: This should help with the spacing of the references.
```suggestion

#page-references article {
    padding: 1vh 2vw 2.5vh 2vw;
}

```

File: src/how-to-place.php [hunk 4]
```diff
@@ -1,17 +1,189 @@
...
+                        </li>
+                        <li>
+                            <b>LA:</b>Left arm. Place symmetrically to RA.
+                        </li>
+                        <li>
+                            <b>RL:</b>Right leg. Place anywhere from the ankle to upper thigh.
+                        </li>
+                        <li>
+                            <b>LA:</b>Left leg. Place symmetrically to RL.
+                        </li>
```
Comment (wintleh) [inline src/how-to-place.php]: Typo?
```suggestion
                        <li>
                            <b>LL:</b>Left leg. Place symmetrically to RL.
                        </li>

```

**Review by wintleh — APPROVED** [review]
Looks good, I included a couple suggestions for the reference spacing

**Pull request merged**